### ISSUE TITLE
Add support for Threads

### ISSUE THREAD

**Issue by Caspy7** [Mte90/Share-Backported#186]
Threads have improved their web presence through the threads.net website.

Please add support so users can create a post to it.

**Comment by Mte90** [comment]
There is a link for threads that let's to share a URL?

The extension use a link provided by the various social networks so if threads still doesn't have it, I can't do anything.

**Comment by Caspy7** [comment]
Does this work now?
https://developers.facebook.com/docs/threads/threads-web-intents

I tested one of the links and it seemed to work.

**Comment by Mte90** [comment]
Yes it is that one, I will do a new release with the social in few minutes.
